components/style/properties/properties.mako.rs

```diff
@@ -1578,10 +1572,10 @@ impl PropertyId {
                 % for (kind, properties) in [("Longhand", data.longhands), ("Shorthand", data.shorthands)]:
                     % for property in properties:
                         "${property.name}" => StaticId::${kind}(${kind}Id::${property.camel_case}),
-                        % for name in property.alias:
-                            "${name}" => {
+                        % for alias in property.alias:
```

Comment:
Wait, the prefixed code does:

```
 property.alias.append('-%s-%s' % (prefix, property.name))
```

So basically there's no object there, how does that work?

Reply:
That happens before we convert the strings into objects.

Reply:
Oh, alright, makes sense.

Thank you, this is a nice cleanup :)